Makefile

```diff
@@ -11,12 +11,23 @@ build:
 	docker-compose build
 
 run:
-	docker-compose up
+	docker-compose up -d
 
 web: run
 
+up: run
+
+down:
+	docker-compose down
+
 migrate:
-	docker exec -it web python manage.py migrate auth || true
-	docker exec -it web python manage.py migrate
+	docker-compose exec web python manage.py migrate auth || true
+	docker-compose exec web python manage.py migrate
+
+migration:
+	docker-compose exec web python manage.py makemigrations --name $(name) $(app)
+
+test:
+	docker-compose exec web python manage.py test $(module)
 
-setup: setup_env build run_db setup_db
+setup: setup_env build run migrate
```

Comment:
`run_db` and `setup_db` are not in the Makefile

Reply:
good! all the options!